docs/src/main/sphinx/connector/hive.rst

```diff
@@ -398,6 +398,8 @@ Property Name                                      Description
                                                    Set to ``false`` to disable statistics. Disabling statistics
                                                    means that :doc:`/optimizer/cost-based-optimizations` can 
                                                    not make smart decisions about the query plan.
+
+``hive.allow-drop-table``                          Enables :doc:`/sql/drop-table` operations.                    ``false``
```

Comment:
This isn't a general Hive property. It should be documented only in the Hive security page.

Reply:
agreed

Reply:
Removed from this section, instead described behavior in more detail down in SQL support